tests/scheduled_checks.py

```diff
@@ -0,0 +1,51 @@
+"""
+These tests are not called on every push or pull request. They're time-intensive, so on GitHub they're only called
+monthly. To run them locally, use the following command:
+
+    pytest -s -rpP tests/scheduled_checks.py
+
+Because the filename doesn't begin with test_, it's not automatically run with pytest. The -s flag is used to print
+the results of the tests, and the -rpP flags are used to suppress the output of the tests that pass.
+
+"""
+from astroquery.simbad import Simbad
+
+from astrodbkit2.utils import _name_formatter
+
+def test_SIMBAD_resolvable(db):
+    # Verify that all sources have valid names
+    results = db.query(db.Sources.c.source).all()
+    name_list = [s[0] for s in results]
+
+    # Add all IDS to the Simbad output as well as the user-provided id
+    Simbad.add_votable_fields("ids")
+    Simbad.add_votable_fields("typed_id")
+
+    simbad_results = Simbad.query_objects(name_list)
+    no_result_rows = []
+    duplicate_rows = []
+    for row in simbad_results[["TYPED_ID", "IDS"]].iterrows():
+        try:
+            name, ids = row[0].decode("utf-8"), row[1].decode("utf-8")
+        except AttributeError:
+            # Catch decoding error
+            name, ids = row[0], row[1]
+
+        simbad_names = [
+            _name_formatter(s)
+            for s in ids.split("|")
+            if _name_formatter(s) != "" and _name_formatter(s) is not None
+        ]
```

Comment:
Is this really necessary?  Isn't there some easy way to tell if the simbad query didn't return any matches?

Reply:
I see, you're using the names below.